bodhi/server/scripts/dequeue_stable.py

```diff
@@ -0,0 +1,40 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""This script is responsible for moving all updates with a batched request to a stable request."""
+
+import sys
+import click
+
+from bodhi.server import config, models, Session, initialize_db
+
+
+@click.command()
+@click.version_option(message='%(version)s')
+def dequeue_stable():
+    """Convert all batched requests to stable requests."""
+    initialize_db(config.config)
+    db = Session()
+
+    try:
+        batched = db.query(models.Update).filter_by(request=models.UpdateRequest.batched).all()
+
+        for update in batched:
+            update.request = models.UpdateRequest.stable
+        db.commit()
+
+    except Exception as e:
+        print(str(e))
+        db.rollback()
+        Session.remove()
+        sys.exit(1)
```

Comment:
This is a fine way to do it, but I'll mention that we also have a [```TransactionalSessionMaker```](https://github.com/fedora-infra/bodhi/blob/2.8.1/bodhi/server/util.py#L664-L683) that you can use to handle the db commit/rollback for you. Here's an [example](https://github.com/fedora-infra/bodhi/blob/2.8.1/bodhi/server/push.py#L65-L67) of using it:

```
initialize_db(config)
db_factory = transactional_session_maker()
with db_factory() as session:
    # Your code goes here, and the db_factory will automatically commit for you if no Exception is raised. It will also
    # automatically rollback if one is. Oh, and the ```session``` it gives you is the same thing as the ```db``` variable
    # you have defined, so that's handy too.
```